google-cloud-tools-plugin/jps-plugin/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2000-2016 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+sourceSets.main.java.srcDirs = ['src']
+sourceSets.main.resources.srcDirs = ['resources']
+sourceSets.test.java.srcDirs = ['testSrc']
+sourceSets.test.resources.srcDirs = ['testData']
+
+jar.archiveName = "google-app-engine-jps-plugin.jar"
+
+// Overriding paths in root build file since this module is nested one level deeper
+pmd {
+    ruleSetFiles = files('../../custom-pmd-ruleset.xml');
```

Comment:
Yea I had meant to ask you about this earlier. This was the best workaround I could come up with, but if I can reference the root directory then I might be able to remove these overrides. I'll try.